test/convertKilometerToMach.test.js

```diff
@@ -0,0 +1,15 @@
+const { convertKilometerToMach } = require("../lib");
+
+describe("convertKilometerToMach", () => {
+  test("1 kilometer is 0.000809848 march", () => {
+    expect(convertKilometerToMach(1)).toBeCloseTo(0.000809848, 4);
+    expect(convertKilometerToMach(5.5)).toBeCloseTo(0.00445416, 4);
```

Comment:
Lets either generalize the `test` description or move each `expect` into it's own `test`block.